tests/models.py

```diff
@@ -7,6 +7,8 @@ class DataToBeScrubbed(models.Model):
     first_name = models.CharField(max_length=8)
     last_name = models.CharField(max_length=255)
     description = models.TextField()
+    ean8 = models.CharField(max_length=13)
+    ean13 = models.CharField(max_length=13)
```

Comment:
this field isn't used?

Reply:
I'll go over that test again. Seems like I messed that up wanting to do just a quick and dirty fix.

Reply:
Removed the surplus field